### PR TITLE
Updated docs in notebook section

### PR DESCRIPTION
**Pull request by Gecervantes01**
Updated the cell collapsing docs

## References

The updated docs are in reference to issue #16179 

## Code changes

The previous documentation on cell collapsing was as follows:
“Collapse and expand code and output using the View menu or the blue
collapser button on left of each cell:”

The suggested updated docs:
“Collapse and expand code and output using the View menu or the blue
collapser button on left of each cell. While collapsed, only the first
line of the cell is displayed:”

This addition at the end provides a clear indication of what is to be expected visually when collapsing a cell.

## User-facing changes

Before:
Before

After:
After

## Backwards-incompatible changes

There are no backwards-incompatible changes.

### CODE REVIEW

**Comment by Gecervantes01** [comment]
bot please update snapshots

File: docs/source/user/notebook.rst [hunk 1]
```diff
@@ -82,7 +82,8 @@ Create multiple synchronized views of a single notebook:
 .. _collapse-code:
 
 Collapse and expand code and output using the View menu or the blue
-collapser button on left of each cell:
+collapser button on left of each cell. While collapsed, only the
+first line of the cell is displayed:
```
Comment (krassowski) [inline docs/source/user/notebook.rst:86]: The video below is well outdated and does not actually show this.

File: galata/test/documentation/general.test.ts-snapshots/interface-right-common-documentation-linux.png
Comment (krassowski) [inline galata/test/documentation/general.test.ts-snapshots/interface-right-common-documentation-linux.png:1]: This is a flaky update and should be reverted. In general, this PR did not change snapshots so far so there is no need to update them.

File: galata/test/jupyterlab/notebook-run.test.ts-snapshots/notebook-panel-1-jupyterlab-linux.png
Comment (krassowski) [inline galata/test/jupyterlab/notebook-run.test.ts-snapshots/notebook-panel-1-jupyterlab-linux.png:1]: This needs to be reverted. https://github.com/jupyterlab/jupyterlab/issues/15305

**Pull request closed without merging**